api/tests/unit/integrations/lead_tracking/hubspot/_hubspot_responses.py

```diff
@@ -0,0 +1,106 @@
+from datetime import datetime
+
+from dateutil.tz import tzlocal
+
+
+class DummyHubspotResponse:
+    """
+    Dummy class to replicate the to_dict() method of the Hubspot response classes.
+    """
```

Comment:
The phrase "Dummy" is frowned upon as it originally and derogatorily means "mute person". See [here](https://github.com/pandas-dev/pandas/issues/35724) for a random discussion about it.

For classes like this I prefer the term `Fake` so `FakeHubspotResponse` in this case.